src/main/java/seedu/taassist/logic/parser/ParserStudentIndexUtil.java

```diff
@@ -0,0 +1,37 @@
+package seedu.taassist.logic.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.taassist.commons.core.Messages;
+import seedu.taassist.commons.core.index.Index;
+import seedu.taassist.logic.parser.exceptions.ParseException;
+import seedu.taassist.model.student.Student;
+
+/**
+ * Contains utility methods used for retrieving students from a list of students based on their indices.
+ * These utility methods are used in the various *Parser classes.
+ */
+public class ParserStudentIndexUtil {
```

Comment:
I think that might not be a good idea. The `ParserUtil` class currently serves the sole purpose of parsing strings. By introducing a new class instead of lumping the methods together, we can maintain better abstraction of the classes. Furthermore, there might be future need of parsing lists when we get to the stage of creating sessions and assigning marks to students. We might be able to modify this class to perform relevant tasks too.